DuckDuckGo/TabViewController.swift

```diff
@@ -1171,6 +1179,8 @@ extension TabViewController: WKNavigationDelegate {
                  completionHandler: @escaping (URLSession.AuthChallengeDisposition, URLCredential?) -> Void) {
         if challenge.protectionSpace.authenticationMethod == NSURLAuthenticationMethodHTTPBasic {
             performBasicHTTPAuthentication(protectionSpace: challenge.protectionSpace, completionHandler: completionHandler)
+        } else if challenge.protectionSpace.authenticationMethod == NSURLAuthenticationMethodServerTrust {
```

Comment:
Since there is a way to initialise the TabViewController from test I think you can test this using something similar to what you can see in macOS in ErrorPageTabExtensionTest